src/Gang/Gang.ts

```diff
@@ -325,7 +325,14 @@ export class Gang {
     if (this.members.length < numFreeMembers && this.respect < Math.pow(recruitCostBase, numFreeMembers)) {
       return numFreeMembers - this.members.length; // if the max possible is less than freeMembers
     }
-    return Math.floor(Math.log(this.respect) / Math.log(recruitCostBase)) + numFreeMembers - this.members.length; //else
+    return (
+      Math.min(
+        Math.floor(Math.log(this.respect) / Math.log(recruitCostBase)),
+        GangConstants.MaximumGangMembers - numFreeMembers,
+      ) +
+      numFreeMembers -
+      this.members.length
+    ); //else
```

Comment:
There's no `if` for this to be an "else" to. (I see this was pre-existing, but it should be killed because it's weird.)

Reply:
Removed